practice/caesar/site.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+var alphabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
+
+/** Takes in a string and shifts it [key] amount on the alphabet to encrypt
+the word.
+*/
+function caesarEncrypt(key, inputWord) {
+  var inputWord = inputWord.toUpperCase();
+  var i = 0;
+  var outputArray = [];
+  for (var x = 0; x < inputWord.length; x += 1) {
+    var inputLetter = inputWord[x];
+    for (var y = 0; y < alphabet.length; y += 1) {
+      var letter = alphabet[y];
+      if (inputLetter === letter) {
```

Comment:
JS arrays have `.indexOf(x)` which will find the index of something for you. Alternatively JS strings have `.charCodeAt(i)` which gives you the ASCII code of a character at a specific index; you can convert that to the index in the alphabet.